RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.0.7
+=====
+* Update instructions of translation process #381
+* Remove test timers related to code block #380
```

Comment:
```suggestion
* Improve test timers handling #380
```

Reply:
This has been addressed in https://github.com/Automattic/studio/pull/412/commits/be5c825387362755911d8fb8897b32a8decc8165.